apollo-converters/moshi/build.gradle

```diff
@@ -3,11 +3,18 @@ apply plugin: 'java'
 targetCompatibility = JavaVersion.VERSION_1_7
 sourceCompatibility = JavaVersion.VERSION_1_7
 
+configurations {
```

Comment:
For android libraries:

https://developer.android.com/studio/build/dependencies.html#library_dependency_configurations